procedure.lua

```diff
@@ -2025,7 +2025,7 @@ function Auxiliary.PendOperation()
 						Duel.Hint(HINT_CARD,0,ce:GetOwner():GetOriginalCode())
 						ce:UseCountLimit(tp)
 					else
-						Duel.SetAdditionalPendulumSummon(tp)
+						Duel.DefaultPendulumSummoned(tp)
```

Comment:
`Duel.UseDefaultPendulumSummon` may be better?

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/2390/commits/89d24fdb888297c861ffbce74ee62d6498dd7de0
renamed.